paasta_tools/cli/cmds/get_latest_deployment.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from paasta_tools.cli.utils import lazy_choices_completer
+from paasta_tools.cli.utils import list_services
+from paasta_tools.cli.utils import PaastaColors
+from paasta_tools.cli.utils import validate_service_name
+from paasta_tools.generate_deployments_for_service import get_latest_deployment_tag
+from paasta_tools.remote_git import list_remote_refs
+from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import get_git_url
+
+
+def add_subparser(subparsers):
+    list_parser = subparsers.add_parser(
+        'get_latest_deployment',
```

Comment:
Can we make this `get-latest-deployment` to be consistent with our other commands?